src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.uap.cs

```diff
@@ -37,7 +40,9 @@ private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args,
             Type t = method.DeclaringType;
             Assembly a = t.GetTypeInfo().Assembly;
 
-            using (AppServiceConnection remoteExecutionService = new AppServiceConnection())
+            IAsyncOperation<AppServiceResponse> asyncOperation = null;
```

Comment:
I did it that way first as a quick test to see if it's working, but then I realized I would like to see any exceptions that occur before sending the message immediately. Figured I would refactor it to do only the wait in a background thread instead of signaling an event and waiting on that, etc.